rewrite-xml/src/main/java/org/openrewrite/xml/tree/Xml.java

```diff
@@ -317,14 +326,14 @@ public Tag withContent(List<? extends Content> content) {
         String beforeTagDelimiterPrefix;
 
         @With
-        Formatting formatting;
+        String prefix;
```

Comment:
Should this be the `Space` class that Jonathan mentioned? Or is that Java-specific?

Reply:
No, `Space` is java specific. XML comments are very constrained in where they can appear in a document, and XML has an AST element for them.